rundeckapp/grails-spa/packages/ui/src/pages/webhooks/i18n.js

```diff
@@ -20,6 +20,11 @@ const translationStrings = {
       webhookUserHelp:'The authorization username assumed when running this webhook. All ACL policies matching this username will apply.',
       webhookRolesLabel:'Roles',
       webhookRolesHelp:'The authorization roles assumed when running this webhook (comma separated). All ACL policies matching these roles will apply.',
+      webhookAuthLabel: 'HTTP Authorization String',
+      webhookGenerateSecurityLabel: "Use Authorization String",
+      webhookGenerateSecretCheckboxHelp: "[Optional] A Webhook Authentication string can be generated to increase security of this webhook. All posts will need to include the generated string in the Authorization header.",
```

Comment:
this messages says "Authentication" but the other messages say "Authorization".  I think they should all say "Authentication"

Reply:
i take that back, even though it is an authentication strring, the actual HTTP header says "Authorization" so let's use "Authorization Header" consistently

Reply:
To help distinguish.... The "feature" will be labeled, documented _Webhook Authentication_.   For the field below was my thinking on calling it Authorization String.

I had asked previously to call it string (instead of secret) to distinguish it from other types of Authorization Headers common in tools like postman etc. where that term shows up.  Trying to make it clear that just the string we generate is the header value and does not include "token" or "secret" as part of the value string.

Reply:
I think for the main ON/Off button it makes sense to call it "Use Authorization Header" as that is what's needed, but maybe the value is still called "Authorization String"?

Reply:
that's fine